notebooks/write_HayabusaNirsIsisLabelNaifSpiceDriver.ipynb

```diff
@@ -0,0 +1,324 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Writing out an ISD from a Hayabusa NIRS image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import json\n",
+    "\n",
+    "import ale\n",
+    "from ale.util import generate_kernels_from_cube\n",
+    "from ale.drivers.hayabusa_drivers import HayabusaNirsIsisLabelNaifSpiceDriver\n",
+    "from ale.formatters.formatter import to_isd\n",
+    "from ale.drivers import AleJsonEncoder\n",
+    "\n",
+    "os.environ['ALESPICEROOT'] = '/Users/jrcain/remotedata/isis_data/'\n",
+    "os.environ['ISISDATA'] = '/Users/jrcain/remotedata/isis_data/'\n",
+    "ale.spice_root = '/Users/jrcain/remotedata/usgs_data/'"
```

Comment:
For compliance when doing releases we have to avoid concrete paths. This also has to be changed in the commit that this is part of, which is frustrating. This is also the case for the KernelSlice notebook.

Reply:
Ah, I missed that for the NIRS notebook.  In the KernelSlice notebook, I put paths like "~/remotedata", so there shouldn't be absolute paths or usernames in there, is that compliant?